js/the_hashtag_generator.js

```diff
@@ -0,0 +1,18 @@
+// https://www.codewars.com/kata/the-hashtag-generator/train/javascript
+
+function generateHashtag(str) {
+  String.prototype.capitalize = function() {
```

Comment:
1. Having this assignment inside a function makes it so that `capitalize` is defined whenever `generateHashtag` is called, essentially overwriting it with a new implementation every time. It is not necessary. It could be defined outside of the function, so that it is set only once 😄 
2. Consider what happens when a new ES feature is added that introduces a `capitalize` method on strings. Let's say that the browser implementation is different from your one. Then, you write some code that depends on the original browser's implementation. Since you have this code here, it breaks the code, as you probably do not expect some unrelated piece of code to overwrite the browser's implementation 😄 

    That is why adding methods to global objects (such as the String's prototype) is discouraged. Having separate functions is generally considered the best solution in such cases 🙂